AdamsToDoConsoleApplication/Task.cs

```diff
@@ -12,22 +12,29 @@ public class Task
         public string mDescription;
         public bool mCompletion;
 
+        public List<taskCollection> Tasks;
+        public class taskCollection
+        {
+            public string mTitle;
+            public string mDescription;
+            public bool mCompletion;
+        }
+
         public Task(string title, string description, bool completion)
```

Comment:
Public properties would be great in here now to start defining between public and private variables.